bcm27xx: bcm27[08-11]: update defconfig files

graysky2 opened the pull request:
PR refreshes kernel defconfig files for all tree subtargets under bcm27xx.

Ran `make kernel_menuconfig CONFIG_TARGET=bcm27yy` having used the snapshot config for each respective target.  For example, for bcm2711 [this]( https://downloads.openwrt.org/snapshots/targets/bcm27xx/bcm2711/config.buildinfo) file.  In order to avoid an error revolving around unsatisfied deps first observed [here](https://github.com/openwrt/openwrt/commit/a478202d74b66c3da17d57442649eb4f131fc7b2#commitcomment-67096592), manually added back two symbols that the make target removed, namely:
```
# CONFIG_SND_SOC_AD193X_I2C is not set
# CONFIG_SND_SOC_AD193X_SPI is not set
```

Another symbol (`# CONFIG_RPI_AXIPERF is not set`) was moved to the generic config thus rendering the changes to the specific configs for both bcm2708 and bcm2709 untouched.

Test building for bcm2711/multidevices (snapshot config file) completed successfully.

graysky2 committed:
bcm27xx: bcm2711: update defconfig

Ran `make kernel_menuconfig CONFIG_TARGET=bcm2711` having used the snapshot
config for bcm2711[1].  Manually added back two symbols that the make target
removed, namely:
* # CONFIG_SND_SOC_AD193X_I2C is not set
* # CONFIG_SND_SOC_AD193X_SPI is not set

Without adding these back, the build fails due to unsatisfied deps[2].

Build system: x86_64
Build-tested: bcm2711/multidevices

1. https://downloads.openwrt.org/snapshots/targets/bcm27xx/bcm2711/config.buildinfo
2. https://github.com/openwrt/openwrt/commit/a478202d74b66c3da17d57442649eb4f131fc7b2#commitcomment-67096592

Signed-off-by: John Audia <graysky@archlinux.us>

graysky2 committed:
bcm27xx: bcm2710: update defconfig

Ran `make kernel_menuconfig CONFIG_TARGET=bcm2710` having used the snapshot
config for bcm2710[1].  Manually added back two symbols that the make target
removed, namely:
* # CONFIG_SND_SOC_AD193X_I2C is not set
* # CONFIG_SND_SOC_AD193X_SPI is not set

1. https://downloads.openwrt.org/snapshots/targets/bcm27xx/bcm2710/config.buildinfo

Signed-off-by: John Audia <graysky@archlinux.us>

graysky2 commented:
@chunkeey - This is my 2nd attempt, previous was https://github.com/openwrt/openwrt/pull/5047

hauke commented on target/linux/bcm27xx/bcm2710/config-5.10:
```diff
@@ -364,7 +363,6 @@ CONFIG_RESET_SIMPLE=y
 CONFIG_RFS_ACCEL=y
 CONFIG_RODATA_FULL_DEFAULT_ENABLED=y
 # CONFIG_RPIVID_MEM is not set
-# CONFIG_RPI_AXIPERF is not set
```
This options should be available when you activate `CONFIG_KERNEL_PERF_EVENTS` in the OpenWrt configuration. Removing it here will probably cause problems when `CONFIG_KERNEL_PERF_EVENTS` is set.

hauke commented on target/linux/bcm27xx/bcm2710/config-5.10:
```diff
@@ -51,6 +49,7 @@ CONFIG_ARM_GIC_V3_ITS_PCI=y
 CONFIG_ARM_PSCI_FW=y
 CONFIG_ARM_RASPBERRYPI_CPUFREQ=y
 CONFIG_ARM_TIMER_SP804=y
+# CONFIG_ASN1 is not set
```
Please move this to target/linux/generic/config-5.10
Since 63f32eb88ee6a4ed96bf28e9b93383e8064a0597 this option is available.

hauke replied:
`# CONFIG_OID_REGISTRY is not set` is already set in target/linux/generic/config-5.10

graysky2 committed:
kernel: move CONFIG_ASN1 to generic config

Rather than populating this symbol in the individual configs, move it to the
generic config.

Signed-off-by: John Audia <graysky@archlinux.us>

hauke commented:
Thank you for the patches, I applied them to master in 4a956a06f23f56162f90eec79a3cbc6caa732681, a68dd8057ce27cb8fe20443fbacfc9b066248133 and d4c20ff948fc9c64b03af648f911f222fcb593a2
